fs2/cli/synthesize.py

```diff
@@ -154,16 +155,23 @@ def prepare_data(
     DEFAULT_LANGUAGE = next(iter(model.lang2id.keys()), None)
     DEFAULT_SPEAKER = next(iter(model.speaker2id.keys()), None)
     if texts:
-        print(f"Processing text {texts}", file=sys.stderr)
-        data = [
-            {
-                "basename": truncate_basename(slugify(text)),
-                text_representation.value: text,
-                "language": language or DEFAULT_LANGUAGE,
-                "speaker": speaker or DEFAULT_SPEAKER,
-            }
-            for text in texts
-        ]
+        data = []
+        for text_input in texts:
```

Comment:
Coverage analysis on codecov.io says none of this code in `prepare_data` is exercised by any unit tests. I guess it's true, since you're testing the callback directly. I think it would be worth adding a direct call to this function, to validate that it's working as expected too.